core/src/main/java/org/hibernate/ogm/jpa/impl/OgmJpaStoredProcedureQuery.java

```diff
@@ -132,7 +122,8 @@ public int getUpdateCount() {
 		return super.getUpdateCount();
 	}
 
-	/* (non-Javadoc)
+	/*
+	 * (non-Javadoc)
 	 * @see org.hibernate.jpa.internal.StoredProcedureQueryImpl#getResultList()
 	 */
```

Comment:
Can you simply remove these comments? As far as I know there is no benefit in keeping this type of comments

Reply:
Same for the other places where it appears